code/game/area/areas/station.dm

```diff
@@ -357,21 +357,25 @@
 /area/station/command/bridge
 	name = "\improper Bridge"
 	icon_state = "bridge"
+	area_flags = ENGINEERING_OVERRIDE_ELIGIBLE
```

Comment:
skyrat edit (or modular file for this whole file

example:
`/area/station/command/bridge
         area_flags = ENGINEERING_OVERRIDE_ELIGIBLE`

Reply:
do this too